exoscale/api/compute.py

```diff
@@ -1313,62 +1406,97 @@ def update(
             None
         """
 
+        kwargs = {}
+
+        if name is not None:
+            kwargs["name"] = name
+
+        if description is not None:
+            kwargs["description"] = description
+
+        if instance_anti_affinity_groups is not None:
+            kwargs["anti-affinity-groups"] = [
+                {"id": i.id} for i in instance_anti_affinity_groups
+            ]
+
+        if instance_deploy_target is not None:
```

Comment:
As we do not use an OpenAPI client, maybe we should benefit from `attrs` metadata to define param name (from API PoV) associated to our class attribute
This way we can generalize this kind of class attributes <-> API params translations